openedx/frontend-app-course-authoring
# fix: combine filter and sort into one modal

JIRA Tickets: [TNL-11055](https://2u-internal.atlassian.net/browse/TNL-11055), [TNL-11069](https://2u-internal.atlassian.net/browse/TNL-11069), [TNL-11032](https://2u-internal.atlassian.net/browse/TNL-11032), [TNL-11168](https://2u-internal.atlassian.net/browse/TNL-11168)

This PR fixes the following:

- Missing filter options
- Filter and sort options not displaying in the same modal
- Filters not displaying to users after they are selected
- File count not updating with filters

Testing

Files page
1. Navigate to the Files page.
2. Click the Sort and filter button
3. Filter options should include: Images, Documents, Code, Audio, Other, Active, Inactive, Public, Locked
4. Select a filter and apply
5. A chip with filter name is shown
6. Click the "x" on the filter chip
7. Chip should be removed
8. Reopen Sort and filter modal
9. Previous filter selection should not be selected.

Videos page
1. Navigate to the Videos page.
2. Click the Sort and filter button
3. Filter options should include: Active, Inactive, Transcribed, Not transcribed, Processing
4. Select a filter and apply
5. A chip with filter name is shown
6. Click the "x" on the filter chip
7. Chip should be removed
8. Reopen Sort and filter modal
9. Previous filter selection should not be selected.
